zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestStandaloneBroker.java

```diff
@@ -70,6 +70,8 @@ protected SpringApplicationBuilder createSpringBuilder() {
     // we need to hook in at the last minute and set the property as it won't resolve from the
     // config bean
     withProperty("zeebe.broker.gateway.enable", config.getGateway().isEnable());
+    withProperty("management.endpoint.prometheus.enabled", true);
```

Comment:
:question: Is this not enabled by default in our `application.properties` file? Do you know why it was necessary to add them here?

Reply:
it is I remember it not working without this but maybe it was due to a different test that used its own config and I assumed that was the base test config